project.clj

```diff
@@ -31,27 +31,27 @@
 
   :profiles {:provided {:dependencies [[org.clojure/clojure "1.7.0"]]}
 
-             :dev {:repl-options {:nrepl-middleware [cider.nrepl.middleware.apropos/wrap-apropos
-                                                     cider.nrepl.middleware.classpath/wrap-classpath
-                                                     cider.nrepl.middleware.complete/wrap-complete
-                                                     cider.nrepl.middleware.debug/wrap-debug
-                                                     cider.nrepl.middleware.enlighten/wrap-enlighten
-                                                     cider.nrepl.middleware.format/wrap-format
-                                                     cider.nrepl.middleware.info/wrap-info
-                                                     cider.nrepl.middleware.inspect/wrap-inspect
-                                                     cider.nrepl.middleware.macroexpand/wrap-macroexpand
-                                                     cider.nrepl.middleware.ns/wrap-ns
-                                                     cider.nrepl.middleware.spec/wrap-spec
-                                                     cider.nrepl.middleware.out/wrap-out
+             :dev {:repl-options {:nrepl-middleware [cider.nrepl/wrap-apropos
```

Comment:
I definitely don't like that all the middleware wrappers ended up being in some utility namespace.